### ISSUE TITLE
[Feature Request] Implement an option to NOT sync the playlist/all data from the logged in account 

### ISSUE THREAD

**Issue by ghost** [DJDoubleD/refreezer#16]
# **🚀 Feature Request**

## **Is your feature request related to a problem? Please describe.**
 
I'm using token from else where and the playlist/fav tracks/artists and everything gets synced

---

## **Describe the solution you'd like**
I need an option where I can opt out of syncing, i.e., playlist I import from Spotify or the songs/album I like should stay locally,it shouldn't sent to remotely and the albums or anything that gets added to actual account holder shouldn't be retrieved in my device.

In short although I do need access for deezer (loll) but not the playlists, artists, albums and all that the account holder created on their end.

---

## **Describe alternatives you've considered**
there's no afaik, I'm just going to stick with Spotify until this gets fixed, this has everything from synced lyrics to hifi downloads, i just wish this gets added soon.

---
